Dockerfile.debian

```diff
@@ -57,8 +57,6 @@ RUN echo "Install OS packages" && \
         socat \
         software-properties-common \
         sudo \
-        systemd \
-        systemd-sysv \
```

Comment:
this change should be reverted we need `systemd` and `systemd-sysv` packages

Reply:
Done